Resources/Prototypes/Recipes/Lathes/misc.yml

```diff
@@ -221,3 +221,21 @@
   completetime: 3
   materials:
     Plastic: 200
+
+- type: latheRecipe
+  id: CassetteTape
+  result: CassetteTape
+  category: Tools
+  completetime: 2
+  materials:
+    Steel: 50
+    Plastic: 150
+
+- type: latheRecipe
+  id: TapeRecorder
+  result: TapeRecorder
+  category: Tools
+  completetime: 3
+  materials:
+    Steel: 250
+    Plastic: 250
```

Comment:
![image](https://github.com/user-attachments/assets/a10e0d41-e08a-4c24-9b8b-3c542d919b94)
Yaml and ftl files shouldn't have this. Fix your editor settings so it linebreaks the last line.